### DISCUSSION TITLE
Build files without content hash, to be able to link from CMS

### DISCUSSION THREAD

**Discussion by kluplau** [vercel/next.js#28162] [Help]
Hi

I'm trying to migrate from CreateReactApp.

- We build the app with our CI tool and deploys it to our static web server.
- We code-split large chunks (features) and link to them directly from a CMS. 
- We can remove hashes now (in Create React App) by disabling the hashing in Webpack. But when I try to do this in Next, it also changes the names of other files than "mine" and it breaks the entire build.

How can I make the chunk naming static, without hashes?

**Comment by azamanaza** [comment]
We're also looking to disabling the hashing without going through hacky implementations in the next.config.js file.

Please please have this as a configurable or optional feature. We have a CMS that requires static names otherwise we have to manually update entrieS, just so it reflects the new file names.
